ext/phook.c

```
@@ -5,9 +5,9 @@
 
 #include "php.h"
 #include "ext/standard/info.h"
-#include "php_opentelemetry.h"
-#include "opentelemetry_arginfo.h"
-#include "otel_observer.h"
+#include "php_phook.h"
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If ZEND_BEGIN_MODULE_GLOBALS is a macro then please configure it.

(unknownMacro)